src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -33,9 +35,13 @@ object Lorre extends App with LazyLogging {
       if (shouldFeeBeCalculatedThisIteration) {
         FeeOperations.processTezosAverageFees()
       }
+      if (shouldAccountsBePurgedThisIteration) {
```

Comment:
Can we not just change the condition to ```iterationsOfLorre % purgeAccountsInterval == 0```?

Reply:
Should "shouldFeeBeCalculatedThisIteration" follow the same treatment, for consistency?

Reply:
Yes, please.

Reply:
On it.